keps/prod-readiness/sig-apps/3088.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3088
+alpha:
+  approver: "@mortent"
```

Comment:
Morten is not a PRR approver. You need to choose from this list:
https://github.com/kubernetes/enhancements/blob/master/OWNERS_ALIASES#L190

I'm happy to take it (but please get SIG approval first).

Reply:
FWIW - you need to fill in the Production Readiness Questionairy from KEP template though.

Reply:
@wojtek-t Thanks for your feedback. 

I changed the approver to you. 

Regarding the SIG approval, the creation of the KEP was proposed is the discussion in the original PR https://github.com/kubernetes/kubernetes/pull/98307 ([comment](https://github.com/kubernetes/kubernetes/pull/98307#issuecomment-771018228), [comment](https://github.com/kubernetes/kubernetes/pull/98307#issuecomment-773050915)). Not sure if it is enough, so I asked in sig/apps slack for the approval. What can I do next?

Reply:
I'd propose starting with sig-apps calls and presenting the idea there, or alternatively start the discussion on sig-apps mailing list to bring the attention.